docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -67,6 +67,37 @@ Renaming a table is a non blocking metadata change operation.
 {{< /note >}}
 
 
+#### SET TABLESPACE *tablespace_name*
+
+Asynchronously change the tablespace of an existing table. 
+The tablespace change will immediately reflect in the config of the table, however the tablet move by the load balancer happens in the background. 
+While the load balancer is performing the move it is perfectly safe from a correctness perspective to do reads and writes, however some query optimization that happens based on the data location may be off while data is being moved.
+
+{{< note title="Note" >}}
+
+This is a BETA feature, and we do not recommend running this command on a production database. You will see a warning when running the command but you can ignore it. Here is an example below:
+
+```sql
+yugabyte=# ALTER TABLE bank_transactions_eu SET TABLESPACE eu_central_1_tablespace;
+WARNING:  'tablespace_alteration' is a beta feature!
+LINE 1: ALTER TABLE bank_transactions_eu SET TABLESPACE eu_central_1...
+                                         ^
+HINT:  To suppress this warning, set the 'ysql_beta_feature_tablespace_alteration' yb-tserver gflag to true.
+(Set 'ysql_beta_features' yb-tserver gflag to true to suppress the warning for all beta features.)
+```
+
+{{< /note >}}
+
+Tables can be moved to the default tablespace using:
+```sql
+ALTER TABLE table_name SET TABLESPACE pg_default;
+```
+
+#### ALTER TABLE ALL IN *tablespace* SET TABLESPACE *tablespace_name*
+
+The `ALL IN TABLESPACE` form will move all tables in the current database into another tablespace. 
+
+
```

Comment:
Remove this section

Reply:
Removed in https://github.com/yugabyte/yugabyte-db/pull/20562/commits/d6063eb00c89837669e795c9bda7f58081997071